fms/models/gpt_bigcode.py

```diff
@@ -193,7 +193,7 @@ def forward(
             # we are caching and can assume all 1s in the mask
             if use_cache and klen != 1 and qlen == 1:
                 # b x h x qlen x kvlen
-                mask = torch.ones(qlen, klen, device=x.device)
+                mask = torch.ones(qlen, klen, device=x.device).bool()
```

Comment:
probably better to `torch.ones(...., dtype=torch.bool)` instead of `torch.ones(...).bool()`. I think the later does a copy to convert from default dtype

Reply:
Thanks! Updated